newsroom/wire/items.py

```diff
@@ -15,6 +16,18 @@ class ItemsResource(BaseItemsResource):
     schema["slugline"] = schema["headline"] = schema["body_html"] = schema["description_html"] = metadata_schema[
         "body_html"
     ].copy()
+    schema["expiry"] = {"type": "datetime", "reaonly": True}
+
+
+class ItemsService(BaseItemsService):
+    def get_expired_items(self, expiry_datetime=None, expiry_days=None, max_results=None, include_children=True):
+        # remove old items based on expiry days config
+        for items in super().get_expired_items(expiry_datetime, expiry_days, max_results, include_children):
```

Comment:
Is it desired that an item will expire if **ANY** of the two conditions are met:
* `_updated` <= expiry_datetime
* `expired` <= expiry_datetime

It seems as though the `SOURCE_EXPIRY_DAYS` is limited up to `CONTENT_API_EXPIRY_DAYS` and cannot be longer than that.

Reply:
changed the core expiry to ignore items with expiry set